packages/core-plugin-api/src/apis/definitions/auth.ts

```diff
@@ -469,3 +469,20 @@ export const vmwareCloudAuthApiRef: ApiRef<
 > = createApiRef({
   id: 'core.auth.vmware-cloud',
 });
+
+/**
+ * Provides authentication towards OpenShift APIs and identities.
+ *
+ * @public
+ * @remarks
+ *
+ * See {@link https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/authentication_and_authorization/configuring-oauth-clients}
+ * on how to configure the OAuth clients and
+ * {@link https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html-single/authentication_and_authorization/index#tokens-scoping-about_configuring-internal-oauth}
+ * for available scopes.
+ */
+export const openshiftAuthApiRef: ApiRef<
```

Comment:
Let's forward this in `frontend-plugin-api` for the new frontend system as well: https://github.com/backstage/backstage/blob/f919be97d995c4eb5cd2b11a48f1daab9809291f/packages/frontend-plugin-api/src/apis/definitions/auth.ts#L17-L40

And the implementation in the `app` plugin: https://github.com/backstage/backstage/blob/0aa9d8275193b7a64588c5ff442da02cfe00c6f3/plugins/app/src/defaultApis.ts#L75